backend/btrixcloud/main.py

```diff
@@ -221,6 +246,31 @@ async def healthz():
 
     app_root.include_router(app, prefix=API_PREFIX)
 
+    # API Configurations -- needed to provide custom favicon
+    @app_root.get(API_PREFIX + "/docs", include_in_schema=False)
+    def overridden_swagger():
+        return get_swagger_ui_html(
+            openapi_url=OPENAPI_URL,
+            title="Browsertrix API",
+            swagger_favicon_url="/favicon.ico",
+        )
+
+    @app_root.get(API_PREFIX + "/redoc", include_in_schema=False)
+    def overridden_redoc():
+        return get_redoc_html(
+            openapi_url=OPENAPI_URL,
+            title="Browsertrix API",
+            redoc_favicon_url="/favicon.ico",
+        )
+
+    def get_api_schema():
+        if not app_root.openapi_schema:
+            app_root.openapi_schema = make_schema()
+
+        return app_root.openapi_schema
+
+    app_root.openapi = get_api_schema  # type: ignore
```

Comment:
This might need to be defined before the docs and redoc endpoints? In local testing, `/api/docs` and `/api/redoc` are failing to load with messages like: `Not Found: /api/openapi.json`.

Reply:
Hm, doesn't seem to be the order, but I'm still seeing:

### http://localhost:30870/api/docs

<img width="604" alt="Screenshot 2024-07-17 at 12 49 49 PM" src="https://github.com/user-attachments/assets/7102f5f4-22ec-49c5-9414-4b25aefb0df1">

### http://localhost:30870/api/redoc

<img width="917" alt="Screenshot 2024-07-17 at 12 53 47 PM" src="https://github.com/user-attachments/assets/4adc5f16-1d5e-406d-86bc-3c995682e6d1">